src/credentials_rotators/npm-token-rotation/src/stacks/base-stack.ts

```diff
@@ -114,4 +110,48 @@ export class BaseStack extends core.Stack {
     });
     alarm.addAlarmAction(new SnsAction(topic));
   };
+
+  protected grandLambdaToAssumeRolePermission = (
```

Comment:
nit, although Lambda is surely grand. :) 

```suggestion
  protected grantLambdaAssumeRolePermission = (
```